src/components/editor/editor-component.ts

```diff
@@ -58,6 +59,11 @@ export const actions = {
     }
   },
 
+  saveTextNote: (nodeId: GraphNodeId) => async (_state: any, _actions: any) => {
+    const data = _state.textEditor.data
+    await writeNote(nodeId, NoteDataType.TEXT, data)
```

Comment:
there no need for the `async` and `await` if you're not waiting for the result of the `writeNote` call.

Reply:
save does not work properly without async and await